RTClib.cpp

```diff
@@ -347,20 +374,32 @@ void RTC_Millis::adjustDrift(int drift) { lastMillis = 1000 - drift; }
 */
 /**************************************************************************/
 DateTime RTC_Millis::now() {
-  uint32_t elapsedSeconds = (millis() - lastMillis) / millisPerSecond;
-  lastMillis += elapsedSeconds * millisPerSecond;
+  uint32_t elapsedSeconds = (millis() - lastMillis) * 1000 / microsPerSecond;
+  lastMillis += elapsedSeconds * microsPerSecond;
```

Comment:
It should be `elapsedSeconds * microsPerSecond / 1000.0`. But then, extra bits would be needed in order to also store the fractional part of `lastMillis`, presumably in a fixed-point format.

Reply:
Fixed by reducing `RTC_Millis` adjust drift precision to parts per thousand. May be increased without overflow error if modulo operator is used. Will investigate in future pull request for `adjustDrift()`